rpm/libmlocale-qt5.spec

```diff
@@ -52,7 +52,9 @@ export QT_SELECT=5
 
 %install
 export QT_SELECT=5
-%make_install
+%qmake5_install
+
+find %{buildroot} -name \*.a -delete
```

Comment:
There's --disable-static above. There shouldn't be anything static?

Reply:
Please check the executed configure file. The original packager of this package assumed the used configure script is an autotools generated script however it is not.
There is no such argument.

Reply:
If there's an argument in the above call that does not exist in the script being called then it should be removed.

Reply:
That exactly ^